website/blog/_guide/content-process.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 2
+unlisted: true
+---
+
+# Content Creation Guide for FerretDB
+
+## Introduction
+
+This guide will walk you through the process of creating and publishing content on the FerretDB blog, which is powered by Docusaurus.
```

Comment:
Team only

Reply:
Should we clarify it somehow? E.g. that contributors are also welcome to write blog posts, but for them the process is simpler.

Reply:
I'm not sure tagging it as simpler would be right though, since they still need to follow almost the same review process, either for topic pitch or completed draft. 

The only difference would be the chance to submit `marketing-oriented` blog as a word document and not as PR. For more technical blog posts, it would follow a similar process. 

This will come in a separate PR, especially with updated information in Contributing_guide.md.

Reply:
Yeah, that sounds reasonable! Thank you for clarifying this!